requirements.in

```diff
@@ -0,0 +1,24 @@
+alembic
+arrow
+bleach
+cached-property
+docutils
+Flask==0.10.1
+Flask-Admin
+Flask-Login==0.2.11
+Flask-Mail
+Flask-Migrate
+Flask-Principal
+Flask-RESTful
+Flask-SQLAlchemy
+Flask-Script
+Flask-Security==1.7.4
```

Comment:
Flask-Security's latest release (1.7.5) contains a number of [bug fixes](https://github.com/mattupstate/flask-security/blob/e01cd63a214969cf8e4ee800d398e1c43b460c7f/CHANGES#L11-L23). This should be upgraded in a separate commit.